packages/stacked_tools/lib/src/templates/compiled_templates.dart

```diff
@@ -1088,6 +1091,40 @@ class {{viewName}} extends StatelessWidget {
 // --------------------------------------------------
 
 
+// -------- GenericView Template Data ----------
+
+const String kViewTemplateGenericViewPath =
+    'lib/ui/views/generic/generic_view.dart.stk';
+
+const String kViewTemplateGenericViewContent = '''
+import 'package:flutter/material.dart';
+import 'package:stacked/stacked.dart';
+
+import '{{viewModelFileName}}';
+
+class {{viewName}} extends ViewModelBuilderWidget<{{viewModelName}}> {
```

Comment:
Refer to the previous PR's review. 

Lets think about a better name for this so it's not too confusing. It should be easy to remember either based on the package we're using or the architecture we're using.